shell/gcloud-pull-staging-files.sh

```diff
@@ -11,3 +11,8 @@ if [ -n "${DEV_NIH_AUTH_ON}" ]; then
   ./google-cloud-sdk/bin/gsutil cp "gs://${GCLOUD_BUCKET}/dev-files/saml/certs/key.pem" ./saml/certs/key.pem
   ./google-cloud-sdk/bin/gsutil cp "gs://${GCLOUD_BUCKET}/dev-files/NIH_FTP.txt" ./NIH_FTP.txt
 fi
+
+# Clone dependencies
+git clone https://github.com/isb-cgc/ISB-CGC-Common.git
```

Comment:
This needs to happen during install-deps. I'll move it after I merge, but I'm pretty certain that's why the builds are breaking now.